test/hive-connect.hcl

```diff
@@ -24,6 +24,28 @@ job "hive" {
       }
     }
 
+    task "waitfor-hive-metastore" {
```

Comment:
```suggestion
    task "waitfor-hive-metastore" {
        restart {
            attempts = 5
            delay    = "30s"
        }
```

What about increasing the restart attempts, so that we don't get failed tasks (considering they haven't actually failed). Right now it fails after two attempts. 
![image](https://user-images.githubusercontent.com/51820995/84120923-c21f2b80-aa36-11ea-8494-506d253040a8.png)

Reply:
10 retries and 15s interval seems OK